src/test/java/utils/WebDriverUtils.java

```diff
@@ -0,0 +1,22 @@
+package utils;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class WebDriverUtils {
+    private static WebDriver driver;
```

Comment:
In case when you use driver wariable only for setWebDriverMethod - no need co keep it on the class level. I suggest to move this variable creation into this method (in case when you don't have any specific reason to keep it here).

Reply:
For now I don't need to have instance variable, moved to the method.